test/support/utils_sycl.h

```diff
@@ -92,13 +93,31 @@ inline auto default_selector =
 #    endif
 #endif     // ONEDPL_FPGA_DEVICE
 
+#if _ONEDPL_DEBUG_SYCL
+static std::once_flag device_name_in_get_test_queue_logged;
+#endif // _ONEDPL_DEBUG_SYCL
+
+template <typename OutputStream>
+inline void
+log_device_name(OutputStream& os, const sycl::queue& queue)
+{
+    os << "    Device Name = " << queue.get_device().template get_info<sycl::info::device::name>() << "\n";
+}
+
 inline
 sycl::queue get_test_queue()
 {
     try
     {
         // create the queue with custom asynchronous exceptions handler
         static sycl::queue my_queue(default_selector, async_handler);
+
+#if _ONEDPL_DEBUG_SYCL
+        std::call_once(device_name_in_get_test_queue_logged, [&]() {
+            log_device_name(std::cout, my_queue);
```

Comment:
Let's remove `log_device_name` and output the information right here. That's a trivial one-line function.

Reply:
>>> That's a trivial one-line function.

- and it's the reason why I declared this function as `inline`.